src/_includes/components/site-logo.html

```diff
@@ -1 +1,3 @@
-<h1>{{ site.title }}</h1>
+<a href="/">
+    <img src="assets/img/canswe-logo-white.png" alt="Canswe Logo" height="100px;">
```

Comment:
Removing the H1 alters the way the document is viewed by the browser, bots, and people who use accessibility tools.

The link can wrap around the existing h1, and use an image-replacement technique to keep the text there, while showing the logo.

The svg version of the logo should be used here.